conftest.py

```diff
@@ -0,0 +1,2 @@
+# make sure hub is config before testing.
+import biothings.hub
```

Comment:
why this `conftest.py` file is needed?

Reply:
This file is a hook which execute before pytest actually run any test file
I added it to make sure the biothings.config must be configured.

If we dont add it, when run tests, an error related to config file will occur:
like this:
```
tests/utils/test_loggers.py:7: in <module>
    from biothings import config
ImportError: cannot import name 'config' from 'biothings'
```

Reply:
Typically the way we get around it is like this:

```
import config    # config typically in the actual hub instance, not the sdk code base. It can be anywhere
biothings.config_for_app(config)
```

After these two lines, `from biothings import config` will work. This like inject `config` module into`biothings` namespace, and `config` module is hub instance specific, should not be part of the sdk.

See a complete example here:

https://github.com/biothings/biothings.species/blob/master/src/bin/hub.py

Reply:
I know, I just thought `import biothings.hub` is more shorter to write.
Anyway, will change code as you suggested :+1:

Reply:
```
(Pdb) import config
(Pdb) import biothings
(Pdb) biothings.config_for_app(config)
(Pdb) from biothings import config
*** ImportError: cannot import name 'config' from 'biothings' (~/biothings.api/biothings/__init__.py)
```

You can see that, even we can run `biothings.config_for_app(config)`, but when we need to import config from biothings, it still failed.

As I checked, the config_for_app method, actually does nothing.
and there are a warning with this message: `It is safe to remove this function call now. import biothings.hub will take care of it.`
and in the `hub._config_for_app`, we do some magic to allow import config

So, I suggest we keep using `import biothings.hub` as I did before.

Reply:
@sengineer0 you are right, that's a simplification I did not know. So `import biothings.hub` will handle it, it's better. 

But where the `conftest` is imported, or should it be part of the test, instead a file in `biothings.api` repo?

Reply:
This file is a global configuration which pytest will automatically import before running any tests.
I placed it at repo's root directory, will move it to the tests module which is better place.

Reply:
Great! Make sense to me now. Please add additional comments and/or link to pytest doc, so that we know how it works in the future.

Reply:
OK. I will add it.